src/site/stages/build/drupal/graphql/healthCareRegionPage.graphql.js

```diff
@@ -8,7 +8,12 @@ const healthCareLocalFacilities = require('./facilities-fragments/healthCareLoca
 const healthCareRegionHealthServices = require('./facilities-fragments/healthCareRegionHealthServices.node.graphql');
 const healthCareRegionNewsStories = require('./facilities-fragments/healthCareRegionNewsStories.node.graphql');
 const healthCareRegionEvents = require('./facilities-fragments/healthCareRegionEvents.node.graphql');
-const healthCareStaffBios = require('./facilities-fragments/healthCareRegionStaffBios.node.graphql');
+// const healthCareStaffBios = require('./facilities-fragments/healthCareRegionStaffBios.node.graphql');
+
+const healthCareStaffBios = requireWithFeatureFlag(
```

Comment:
I feel a little dirty making this a global. We could make it so you have to do a `require` here for `requireWithFeatureFlag`.

Reply:
Yeah, global seems fine to me. Requiring a special require seems a little funny lol.